package.json

```diff
@@ -16,10 +16,14 @@
   "dependencies": {
     "material-design-icons-iconfont": "^5.0.1",
     "vue": "^2.6.10",
-    "vuetify": "^1.5.16"
+    "vuetify": "^1.5.16",
+    "highcharts": "^7.1.2",
```

Comment:
Why including that at the root level? Isn't it possible to include that just for `components/Chart` package?

Reply:
I don't sure about this but I will try with the package in ```components/Chart```